### PR TITLE
Feature/typing and recompose

### PR DESCRIPTION
**Pull request by mdvanes**

### CODE REVIEW

File: src/EventsHeader/DetailsSection.test.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { DetailsSection } from './DetailsSection';
+import renderer from 'react-test-renderer';
+
+const t = () => '';
```
Comment (mvdam) [inline src/EventsHeader/DetailsSection.test.tsx]: What does this represent? 😁

File: src/components/Events/Events.tsx [hunk 2]
```diff
@@ -16,56 +13,55 @@ type EventsPropTypes = any;
...
-		noNextMeetupEvent: PropTypes.bool.isRequired,
-		pastMeetupEvents: PropTypes.array.isRequired,
-	};
+// TODO improve types by replacing "any"
+interface IEventProps {
+	t: TranslationFunction;
+	nextMeetupEvents: any[];
+	noNextMeetupEvent: boolean;
+	pastMeetupEvents: any[];
+}
```
Comment (mvdam) [inline src/components/Events/Events.tsx]: 👍 good to see the proptypes go!

File: src/EventsHeader/DetailsSection.test.tsx (hunk 1 above)
Comment (mdvanes) [reply src/EventsHeader/DetailsSection.test.tsx]: This is how you can mock `t` (the translate function in react-i18next) now this is a Stateless Functional Component.

Comment (mvdam) [reply src/EventsHeader/DetailsSection.test.tsx]: Ah good to know. Maybe add a comment about this?

Comment (mdvanes) [reply src/EventsHeader/DetailsSection.test.tsx]: Done!

**Pull request merged**